src/Cache/Cache.php

```diff
@@ -239,7 +239,7 @@ public static function pool(string $config)
      * @param string $config Optional string configuration name to write to. Defaults to 'default'
      * @return bool True if the data was successfully cached, false on failure
      */
-    public static function write(string $key, $value, string $config = 'default'): bool
+    public static function write(string $key, mixed $value, string $config = 'default'): bool
```

Comment:
Is mixed a native type now?

Reply:
With php 8, we get a few new types:

- mixed
- false
- static

static can only be used a return type and false cannot be used by itself, must be part of a union.

Reply:
We also get an interface called `Stringable`. https://www.php.net/manual/en/class.stringable